src/bin/vip-dev-env-import-sql.js

```diff
@@ -43,6 +43,7 @@ command( {
 	.option( 'slug', 'Custom name of the dev environment' )
 	.option( 'search-replace', 'Perform Search and Replace on the specified SQL file' )
 	.option( 'in-place', 'Search and Replace explicitly on the given input file' )
+	.option( 'add-vipgo-user', 'Add vipgo:password user to the site to enable easy access' )
```

Comment:
I am not sure if we should do this conditionally as an option or just always add this user. What do you think?

Reply:
I think always, TBH. I actually stumbled upon it last night and thought "it'd be nice to keep the user". How are you reading my mind?

Reply:
Done.